src/python/bin/aqueduct

```diff
@@ -142,13 +128,21 @@ def update_server_version():
 
     execute_command([os.path.join(server_directory, "bin", "migrator"), "--type", "sqlite", "goto", "8"])
 
-def generate_welcome_message(expose_ip):
-    if not expose_ip:
+def generate_welcome_message(expose):
+    if not expose:
         expose_ip = "localhost"
+    else:
+        expose_ip = "<PUBLIC_IP>"
```

Comment:
What's going on here?

Reply:
@vsreekanti Since we don't require the user to input the ip anymore, we don't know the public ip right? So I'm putting a placeholder here. Any better ideas?